com.unity.multiplayer.mlapi/Tests/Runtime/Metrics/Messaging/NetworkMetricsMessagingDispatchTests.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace MLAPI.RuntimeTests.Metrics.Messaging
 {
-#if true
+#if MULTIPLAYER_TOOLS
```

Comment:
So would we consider it to be a normal situation to be where we could be running MLAPI tests but can't run these because the tools library is present?

If so, ok, but if not then this ifdef as is would just mask the issue and we wouldn't run tests we deem are essential for passing MLAPI.  If I have this right so far, would we want to have it where tools isn't required for this to compile, but the lack of tools would cause the test execution to blow up?

Reply:
The way this is defined, those tests just don't exist if the tools library is not present.

Either way, whether the library is there or not, the tests are green.

I think the question is: do we want to run these tests with or without (or both!) in our CI pipeline.

Reply:
Discussed in Slack, the output is:

- Runtime has the lib conditional, for users who want to opt-out
- Tests are just for us, so always require the lib, remove conditional define since we want tests to break if the lib is not present